fiftyone/core/config.py

```diff
@@ -154,6 +154,12 @@ def __init__(self, d=None):
             env_var="FIFTYONE_DEFAULT_BATCH_SIZE",
             default=None,
         )
+        self.bulk_write_batch_size = self.parse_int(
+            d,
+            "bulk_write_batch_size",
+            env_var="FIFTYONE_BULK_WRITE_BATCH_SIZE",
+            default=100_000,  # mongodb limit
```

Comment:
never realized `100_000` was valid syntax 🤯

Reply:
Yes I recently discovered it, very useful for readability. Works with any numeric literal like `0xCAFE_BEEF`.  It's in python 3.6 so no worries about support.